src/MSBuild.Abstractions/MSBuild.Abstractions.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
This prevents us from running on the 3.1 runtime. I don't see a reason why someone should require the 5.0 runtime to do conversions. Can't we keep this as 3.1?